librz/include/rz_core.h

```diff
@@ -548,6 +548,17 @@ RZ_API ut64 rz_core_get_asmqjmps(RzCore *core, const char *str);
 RZ_API void rz_core_set_asmqjmps(RzCore *core, char *str, size_t len, int i);
 RZ_API char *rz_core_add_asmqjmp(RzCore *core, ut64 addr);
 
+typedef struct analysis_bytes_t {
```

Comment:
I think it belongs in `librz/include/rz_analysis.h`, please also add Doxygen.

Reply:
oh, i did not notice, yes, place it in rz_analysis.h